easybuild/easyblocks/g/gcc.py

```diff
@@ -1043,4 +1048,42 @@ def make_module_req_guess(self):
             'LD_LIBRARY_PATH': ['lib', 'lib64'],
             'MANPATH': ['man', 'share/man']
         })
+
+        # Install rpath wrappers, if built with RPATH enabled
+        if build_option('rpath'):
+            def create_rpath_wrappers(targetdir):
+                set_up_configuration(silent=True)
+                tc = Toolchain(name="system", version="0.1")
```

Comment:
Although it probably doesn't matter much, we should use `GCCcore` as name, and `self.version` as version here to create the toolchain instance:

```python
tc = Toolchain(name=self.name, version=self.name)

Reply:
Following our discussion, I'm going the `get_toolchain` route now to correctly setup the `self.CC_COMPILER` variables etc.